MD/md-26/README.md

```diff
@@ -0,0 +1,87 @@
+# MD-26: User-facing checks
+- **Description**: Automate checking user experience by submitting transactions every few minutes
+- **Authors**: [Andy Golay](mailto:liam@movementlabs.xyz)
+
+
+<!--
+  This template is for drafting Desiderata. It ensures a structured representation of wishes, requirements, or needs related to the overarching objective mentioned in the title. After filling in the requisite fields, please delete these comments.
+
+  Note that an MD number will be assigned by an editor. When opening a pull request to submit your MD, please use an abbreviated title in the filename, `md-draft_title_abbrev.md`.
+
+  TODO: Remove this comment before finalizing.
+-->
+
+## Overview
+We need to check that users can submit transactions. This MD outlines a simple automated solution.
+
+## Definitions
+
+Provide definitions that you think will empower the reader to quickly dive into the topic.
+
+## Desiderata
+
+  ### Desideratum Title
+
+  **User Journey**: Users get more assurance that we know when Suzuka testnet has a transaction-processing outage. 
+
+  **Description**:
+
+  1. Spin up an EC2 instance.
+  
+  2. On the instance, create a shell script `suzuka_user_test.sh`:
```

Comment:
Instead of creating additional difficult-to-maintain bash scripts, I would suggest reusing e2e tests and thus making this desideratum some like:

"Node operators and community can observe network health by conditions specified by e2e tests."

Then your current user journey would be part of the justification and a corresponding MIP would identify the e2e tests to be included.

Reply:
We have e2e tests which are already packaged as containers. See https://github.com/movementlabsxyz/movement/tree/main/networks/suzuka/suzuka-client/src/bin/e2e for the test logic and https://github.com/orgs/movementlabsxyz/packages?repo_name=movement for the containers (those with e2e in the name).

Reply:
I would further suggest it should be the responsibility of the monitoring implementation to leverage these binaries and containers, i.e., reporting based on the exit code of the e2e test binaries and not requiring any modifications to the implementation.

Reply:
`simple_interaction.rs` looks perfect for this. 

When I pull and run the container I get

```
thread 'main' panicked at networks/suzuka/suzuka-client/src/bin/e2e/simple_interaction.rs:12:66:
called `Result::unwrap()` on an `Err` value: Dot movement path not provided
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

- Does there need to be some additional configuration other than just pulling and running the container?

- I noticed the `simple_interaction.rs` has a one-time simple interaction. What's a good way to run it repeatedly?Call `docker run` every few minutes? Or is there a better way that you recommend?

Reply:
You need to provide a a DOT_MOVEMENT_PATH environment variable which would point to a correctly configure dot movement config.

Reply:
@l-monninger updated per your change request and pinged for re-review